cookbooks/aws-parallelcluster-config/attributes/default.rb

```diff
@@ -1,3 +1 @@
 # aws-parallelcluster-config attributes
-
-default['cluster']['scheduler_slots'] = 'vcpus'
```

Comment:
you can delete this file, it's empty now!

Reply:
Cool wasnt sure whether I should.